examples/set-annotation/complete-config/.expected/diff.patch

```diff
@@ -1,7 +1,7 @@
-diff --git a/functions/fn-config.yaml b/functions/fn-config.yaml
+diff --git a/fn-config.yaml b/fn-config.yaml
```

Comment:
Re. directory name, does it make sense to change `examples/set-annotation/complete-config/` to for example `examples/set-annotation/advanced/`?

This comment also applies to `set-labels`.

Reply:
updated